WcaOnRails/app/models/registration.rb

```diff
@@ -192,4 +192,24 @@ def events_to_associated_events(events)
       registration_competition_events.find_by_competition_event_id(competition_event.id) || registration_competition_events.build(competition_event: competition_event)
     end
   end
+
+  def serializable_hash(options = {})
+    json = {
+      class: self.class.to_s.downcase,
+      id: id,
+      competition_id: competition_id,
+      user_id: user_id,
+      event_ids: events.map { |e| e.id },
```

Comment:
Same here, another possibility is to include an `Event` objet, which would basically contain the event id and its name in English.
Since registrations list can be quite big, I opted for the id-only way.